### PR TITLE
Move Jobs updated time display logic to client.

### PR DESCRIPTION
**Pull request by jmchilton**
Displays actual time now on mouseover in addition to elapsed time now.

I want this for two reasons - first to eliminate the the admin/jobs web controller, we really should be using the jobs API for this component. This brings us closer to having these two controllers in sync (still quite a bit work to do I'm sure). Second, I want to have a date display component for the invocations grid after merge of #8699 - the time display in those grids is very rough currently.

### CODE REVIEW

File: client/galaxy/scripts/components/UtcDate.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            type: String,
+            required: true
+        },
+        mode: {
+            type: String,
+            default: "date" // or elapsed
+        }
+    },
+    computed: {
+        elapsedTime: function() {
```
Comment (dannon) [inline client/galaxy/scripts/components/UtcDate.vue:25]: We have moment as dependency right now (due to handsontable, so if we're changing that out this is moot), and if you wanted to use that it'd simplify the bits below quite a bit.

(not a required or even requested change, just throwing this out there for thought)

**Comment by Nerdinacan** [comment]
Would this be better implemented as a directive or a filter perhaps?

```html
<span>{{ someRawDate | utcdate }}</span>
<span v-utcdate>{{ rawDate }}</span>
```

Or maybe as just a model property? I think I had a property of my client-side model container that did this on the history model at one point or another.

```javascript
class Job {

    get utcDate() {
        return moment.utc(this.update_time);
    }
}
```

File: client/galaxy/scripts/components/UtcDate.vue (hunk 1 above)
Comment (martenson) [reply client/galaxy/scripts/components/UtcDate.vue:25]: :+1: for momentjs, pure JS is horrendous with dates

**Review by martenson — APPROVED** [review]

**Comment by martenson** [comment]
@Nerdinacan the current implementation feels like a pretty clean solution to me, especially since this will hopefully be reused often. Do you still think it'd be better implemented differently?

**Pull request merged**